### PR TITLE
add direction to css and page template

### PR DESCRIPTION
**Pull request by ghaseminya**
Hi @ruanyf 
I change some needed css tag in app.css and page.template to accept new config file for rtl support.

### CODE REVIEW

File: theme/page.template [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <![endif]-->
 <link rel="shortcut icon" href="">
 </head>
-<body>
+<body dir="<%= direction %>">
```
Comment (ruanyf) [inline theme/page.template:18]: Did you test it?

After adding `dir="RTL"` into the page, I got the following looking. Is it right?

redux documents

Comment (ghaseminya) [reply theme/page.template:18]: Yes, This type setting is true and complete, but in script mode must be left to right that i change app.css for this result.
I am writing new book with this new change in Persian language,

Comment (ghaseminya) [reply theme/page.template:18]: screenshot from 2017-04-08 01-24-16

Comment (ghaseminya) [reply theme/page.template:18]: https://ghaseminya.github.io/j2ee-farsi/

**Pull request merged**